test/integration/usersFilter.test.js

```diff
@@ -0,0 +1,296 @@
+const { generateUserStatusData } = require("../fixtures/userStatus/userStatus");
+
+const config = require("config");
+const { updateUserStatus } = require("../../models/userStatus");
+const { addTag } = require("../../models/tags");
+const { addLevel } = require("../../models/levels");
+const { addTagsToItem } = require("../../models/items");
+const { assertUserIds } = require("../utils/user");
+
+const cookieName = config.get("userToken.cookieName");
+
+chai.use(chaiHttp);
+
+describe("Filter Users", function () {
+  let jwt;
+  let userId = "";
+  let oooUser;
+  let idleUser;
+  let activeUser;
+  let tagIdFE;
+  let tagIdBE;
+  let levelId1;
+  let levelId2;
+
+  before(async function () {
+    const updatedAtDate = Date.now();
+    const untilDate = updatedAtDate + 16 * 24 * 60 * 60 * 1000;
+    userId = await addUser();
+    jwt = authService.generateAuthToken({ userId });
+    oooUser = await addUser(userData[0]);
+    await updateUserStatus(
+      oooUser,
+      generateUserStatusData("OOO", updatedAtDate, updatedAtDate, untilDate, "Bad Health")
+    );
+    idleUser = await addUser(userData[1]);
+    await updateUserStatus(idleUser, generateUserStatusData("IDLE", updatedAtDate, updatedAtDate, untilDate, "CSS"));
+    activeUser = await addUser(userData[8]);
+    await updateUserStatus(activeUser, generateUserStatusData("ACTIVE", updatedAtDate, updatedAtDate));
+
+    // creating tag and levels
+    const { id: id1 } = await addTag({
+      name: "Vue JS",
+      type: "SKILL",
+      reason: "Web Dev FE",
+    });
+    tagIdFE = id1;
+    const { id: id2 } = await addTag({
+      name: "Node JS",
+      type: "SKILL",
+      reason: "Web Dev BE",
+    });
+    tagIdBE = id2;
+    const { id: id3 } = await addLevel({
+      name: "level 1",
+      value: 1,
+    });
+    levelId1 = id3;
+    const { id: id4 } = await addLevel({
+      name: "level 2",
+      value: 2,
+    });
+    levelId2 = id4;
+
+    // creating items
+    await addTagsToItem({
+      itemId: oooUser,
+      itemType: "USER",
+      tagPayload: [
+        {
+          tagId: tagIdFE,
+          levelId: levelId1,
+        },
+        {
+          tagId: tagIdFE,
+          levelId: levelId2,
+        },
+      ],
+    });
+    await addTagsToItem({
+      itemId: idleUser,
+      itemType: "USER",
+      tagPayload: [
+        {
+          tagId: tagIdBE,
+          levelId: levelId1,
+        },
+        {
+          tagId: tagIdBE,
+          levelId: levelId2,
+        },
+      ],
+    });
+    await addTagsToItem({
+      itemId: activeUser,
+      itemType: "USER",
+      tagPayload: [
+        {
+          tagId: tagIdFE,
+          levelId: levelId1,
+        },
+        {
+          tagId: tagIdBE,
+          levelId: levelId1,
+        },
+      ],
+    });
+  });
+
+  after(async function () {
+    await cleanDb();
+  });
+
+  // eslint-disable-next-line mocha/no-skipped-tests
+  describe.skip("GET /users/search", function () {
```

Comment:
shouldnt we remove this `.skip` here

Reply:
If we remove the `skip` then the tests will start failing. Once this gets merged then we can remove it.